internal/storage/v2/grpc/client.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package grpc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"iter"
+
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"google.golang.org/grpc"
+
+	"github.com/jaegertracing/jaeger/proto-gen/storage/v2"
+)
+
+type Client struct {
+	readerClient storage.TraceReaderClient
+}
+
+func NewClient(tracedConn *grpc.ClientConn) *Client {
+	return &Client{
+		readerClient: storage.NewTraceReaderClient(tracedConn),
+	}
+}
+
+func (c *Client) GetTraces(ctx context.Context,
+	traceIDs ...storage.GetTraceParams,
```

Comment:
I think this struct should be implementing TraceReader API, so should it not be taking `tracestore.GetTraceParams`?